ringo_printtemplate/views.py

```diff
@@ -126,9 +129,13 @@ def _render_template(request, template, item):
     """
     out = StringIO.StringIO()
     data = load_file(template.data)
-    temp = Template(StringIO.StringIO(data), out)
-    temp.render({"item": item, "print_item": PrintValueGetter(item, request)})
-    return out
+    mime = check_mime_type_from_buffer(data)
+    print(mime)
+    if mime == "application/vnd.oasis.opendocument.text":
+        temp = Template(StringIO.StringIO(data), out)
+        temp.render({"item": item, "print_item": PrintValueGetter(item, request)})
+        return out
+    return None
```

Comment:
Ok. I am fine with this change. IMO this check is not really needed but cheap enough to do this check on every read.